hack/local-up-cluster.sh

```diff
@@ -30,6 +30,7 @@ RUNTIME_CONFIG=${RUNTIME_CONFIG:-""}
 KUBELET_AUTHORIZATION_WEBHOOK=${KUBELET_AUTHORIZATION_WEBHOOK:-""}
 KUBELET_AUTHENTICATION_WEBHOOK=${KUBELET_AUTHENTICATION_WEBHOOK:-""}
 POD_MANIFEST_PATH=${POD_MANIFEST_PATH:-"/var/run/kubernetes/static-pods"}
+KUBELET_IMAGE={KUBELET_IMAGE:-"k8s.gcr.io/hyperkube-amd64:latest"}
```

Comment:
The other thing is.. are you looking for an image that exists? The `latest` tag doesn't exist for the production image.

Reply:
Guess it would be better to leave ":latest" off